load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -1034,148 +1033,195 @@ func (s *relaysSuite) sendRelay(iteration uint64, relayPayload string) (appAddre
 	gateway := s.activeGateways[iteration%uint64(len(s.activeGateways))]
 	application := s.activeApplications[iteration%uint64(len(s.activeApplications))]
 
-	gatewayUrl, err := url.Parse(s.gatewayUrls[gateway.address])
-	require.NoError(s, err)
-
-	// Include the application address in the query to the gateway.
-	query := gatewayUrl.Query()
-	query.Add("applicationAddr", application.address)
-	query.Add("relayCount", fmt.Sprintf("%d", iteration))
-	gatewayUrl.RawQuery = query.Encode()
-
-	// Use the pre-defined service ID that all application and suppliers are staking for.
-	gatewayUrl.Path = testedServiceId
-
 	// TODO_MAINNET: Capture the relay response to check for failing relays.
 	// Send the relay request within a goroutine to avoid blocking the test batches
 	// when suppliers or gateways are unresponsive.
-	go func(gwURL, payload string) {
-		_, err = http.DefaultClient.Post(
-			gwURL,
-			"application/json",
-			strings.NewReader(payload),
-		)
+	sendRelayRequest := func(gatewayURL, appAddr, payload string) {
+		gatewayURL = gatewayURL + "/"
```

Comment:
1. #PUC why we need to add "/"
2. Should we not check if there's a trailing / first? Maybe the std string library has something for it